src/Symfony/Component/Validator/Resources/translations/validators.de.xlf

```diff
@@ -338,6 +338,10 @@
                 <source>This collection should contain only unique elements.</source>
                 <target>Diese Sammlung darf keine doppelten Elemente enthalten.</target>
             </trans-unit>
+            <trans-unit id="88">
+                <source>This password has been leaked in a data breach, it must not be used. Please use another password.</source>
+                <target>Dieses Passwort ist Teil eines Datenlecks, es kann darf nicht verwendet werden. Bitte benutze ein anderes Passwort.</target>
```

Comment:
kann darf

?? kann/darf oder nur darf?

Reply:
just "darf"

Reply:
I would leave it as it is. We forbid to use that password, so „darf“ would be correct.